### ISSUE TITLE
Address 311 <> Knack Integration Issues

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#5912]
We're experiencing frequent downtime of our 311 <> Knack integration.

After thorough troubleshooting, we believe the issues stem from the Enterprise Service Bus (ESB), which is a CTM-managed tool that sends records from 311 to our [Knack proxy server](https://github.com/cityofaustin/atd-knack-proxy).

**Comment by johnclary** [comment]
Sample error message:
```
Server:  <ESB production>

Application: Knack [CSR]
Flow: Handle_CSR_Event

Message returned to queue for reprocessing.

========== Error Details ==========

A Web Service request has detected a SOCKET error whilst invoking a web service located at host &1, on port &2, on path &3.

Host: knack-proxy.austinmobility.io
Port: 443
Path: /v1/objects/object_83/records

An error occurred whilst performing an SSL socket operation

connect

java.net.ConnectException: Connection timed out: connect

========== Knack ==========

**Comment by johnclary** [comment]
Sergio, Mateo and I met with Omar and spent time reviewing our AWS Lambda and API Gateway configuration to think through issues/solutions. Sergio activated `ERROR` level logging on the API gateway. I conducted load tests against our Knackpy Development app. It is smaller/more performant than the Data Tracker, but I encountered no issues whatsoever while inserting 3k records on 16 concurrent threads.

At this time we're going to have to wait until the issue crops up again. If/when that happens we should review the API gateway logs to see if any requests are being dropped.
